modules/gcloud/bigquery.go

```diff
@@ -25,20 +25,20 @@ func RunBigQueryContainer(ctx context.Context, opts ...testcontainers.ContainerC
 		return nil, err
 	}
 
-	req.Cmd = []string{"--project", settings.ProjectID}
+	req.Cmd = append(req.Cmd, "--project", settings.ProjectID)
 
 	container, err := testcontainers.GenericContainer(ctx, req)
 	if err != nil {
 		return nil, err
 	}
 
-	spannerContainer, err := newGCloudContainer(ctx, 9050, container, settings)
+	bigqueryContainer, err := newGCloudContainer(ctx, 9050, container, settings)
```

Comment:
Looks like this needs a rebase as there has been some work to ensure this correctly returns the container even on error.